Lib/test/test_zoneinfo/test_zoneinfo.py

```diff
@@ -45,6 +45,11 @@ def setUpModule():
     global ZONEINFO_DATA
     global ZONEINFO_DATA_V1
 
+    try:
+        import lzma
+    except ImportError:
+        raise unittest.skip("Failed to import lzma.")
```

Comment:
```suggestion
        raise unittest.SkipTest("Failed to import lzma.")
```

`SkipTest` is the appropriate exception to be raised. Using `skip` causes the below error.

```
./python -Wall -m test test_zoneinfo
0:00:00 load avg: 0.03 Run tests sequentially
0:00:00 load avg: 0.03 [1/1] test_zoneinfo
test test_zoneinfo crashed -- Traceback (most recent call last):
  File "/root/cpython/Lib/test/test_zoneinfo/test_zoneinfo.py", line 26, in <module>
    import lzma
  File "/root/cpython/Lib/lzma.py", line 27, in <module>
    from _lzma import *
ModuleNotFoundError: No module named '_lzma'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/root/cpython/Lib/test/libregrtest/runtest.py", line 272, in _runtest_inner
    refleak = _runtest_inner2(ns, test_name)
  File "/root/cpython/Lib/test/libregrtest/runtest.py", line 223, in _runtest_inner2
    the_module = importlib.import_module(abstest)
  File "/root/cpython/Lib/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 790, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/root/cpython/Lib/test/test_zoneinfo/__init__.py", line 1, in <module>
    from .test_zoneinfo import *
  File "/root/cpython/Lib/test/test_zoneinfo/test_zoneinfo.py", line 28, in <module>
    raise unittest.skip("Failed to import lzma.")
TypeError: exceptions must derive from BaseException

test_zoneinfo failed

== Tests result: FAILURE ==

1 test failed:
    test_zoneinfo

Total duration: 60 ms
Tests result: FAILURE
```

Reply:
Actually, @vstinner made a better suggestion in [bpo-41475](https://bugs.python.org/issue41476), using [`test.support.import_module`](https://docs.python.org/3/library/test.html#test.support.import_module) to import `lzma`: 

That skips the tests if the import fails.

Reply:
Ill update the PR later today. Thank you for the feedback!